src/Components/Generator/SignatureInput.js

```diff
@@ -61,8 +68,10 @@ export default class SignatureInput extends preact.Component {
             // see https://stackoverflow.com/a/4776378
             let img = new Image();
             img.onload = () => {
+                const left = (this.canvas.width - img.width) / 2;
+                const top = (this.canvas.height - img.height) / 2;
```

Comment:
Since you are already working with the image, please also set the `cropArea` to the area of the image. Otherwise the image will just be cropped away if the user doesn't draw a circle around it (defeating the purpose of prefilling).